runtime/onert/core/src/ir/train/TrainableGraph.cc

```diff
@@ -64,6 +64,11 @@ IOIndex TrainableGraph::getOutputIndex(const std::string &name) const
   return _graph.getOutputIndex(name);
 }
 
+void TrainableGraph::changeShape(const OperandIndex &index, const ir::Shape &new_shape)
+{
+  _graph.changeShape(index, new_shape);
+}
+
```

Comment:
@ragmani I think it is not the scope of this PR. Please let me know if it needs to be included.

Reply:
It is the scope of this PR. I forgot to add the definition of this virtual method in `TrainableGraph`.

Reply:
I thought this PR fixes the missing `override` **keyword** in TrainableGraph. But I found the title is "Fix missing override **methods** in TrainableGraph". `changeShape` is the missing method. :)